OpenSim/Simulation/SimbodyEngine/Joint.h

```diff
@@ -33,6 +33,24 @@ namespace OpenSim {
 
 class Model;
 
+class JointCannotJoinTheSamePhysicalFrame : public Exception {
+public:
+    JointCannotJoinTheSamePhysicalFrame(const std::string& file,
+        size_t line,
+        const std::string& func,
+        const std::string& thisName,
+        const std::string& parentName,
+        const std::string& childName,
+        const std::string& baseName) :
+        Exception(file, line, func) {
+        std::string msg = "Joint '" + thisName + "' attempting to connect '" +
```

Comment:
Suggest:
attempting to -> cannot
on the same... -> because they have the same...